syft/workers/websocket_client.py

```diff
@@ -38,7 +38,8 @@ def __init__(
 
         # creates the connection with the server which gets held open until the
         # WebsocketClientWorker is garbage collected.
-        self.ws = websocket.create_connection(self.uri, max_size=None)
+        # Daniele Gadler: Also avoid the server from timing out on the server-side
```

Comment:
@DanyEle can you remove the author name from the line? It's not a code pattern in syft to keep the author's name in a comment.

Reply:
Done!